tox.ini

```
@@ -0,0 +1,6 @@
+[tox]
+envlist = python3.7,python3.8,python3.9,python3.10
```

To support different python versions with tox you may use a dependency matrix:
https://tox.wiki/en/latest/example/basic.html#compressing-dependency-matrix

```suggestion
envlist = py{37,38,39,310}
```